README.md

```diff
@@ -40,6 +40,14 @@ Build fonts to edit TrueType instructions in VTT (placed next to the UFO)
 $ make vtt
 ```
 
+When working on the fonts, "normalize" them before you check them in. Tools
+tend to format the underlying files however they want, leading to large,
+pointless differences to what's in the repository.
+```
+$ pip install ufonormalize # Install this first.
```

Comment:
It’s ufonormalizer with the r
Why not adding it to requirements.txt?

Reply:
Oops.

Hm, it's not strictly required to build stuff, but I'm not fond of adding a dev-requirements.txt either. Oh well, will add.